src/components/FormTextField/FormTextField.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from "@testing-library/react-native"
+import FormTextField from "./FormTextField"
+
+describe("FormTextField component", () => {
+  it("renders label", () => {
+    render(<FormTextField label="Hello!" value="response"></FormTextField>)
+    expect(screen.getByText(/Hello/)).toBeOnTheScreen()
+  })
```

Comment:
```suggestion
  it("renders label", () => {
    const handleChangeMock = jest.fn()
    render(<FormTextField label="Hello!" value="response" onChange={handleChangeMock}></FormTextField>)
    expect(screen.getByText(/Hello/)).toBeOnTheScreen()
    user.keyboard(screen.getByLabelText(/Hello/i), "test")
    expect(handleChangeMock).toHaveBeenCalledWith("test")
  })
```

Could use a test on the change event, and its probably something like that.